source/patterns/@aws-solutions-constructs/aws-cloudfront-s3/test/test.cloudfront-s3.test.ts

```diff
@@ -313,4 +313,45 @@ test('Cloudfront logging bucket error when providing existing log bucket and log
   };
 
   expect(app).toThrowError();
+});
+
+// --------------------------------------------------
+// CloudFront origin path
+// --------------------------------------------------
+test('CloudFront origin path present when provided', () => {
+  const stack = new cdk.Stack();
+
+  new CloudFrontToS3(stack, 'cloudfront-s3', {
+    originPath: '/testPath'
+  });
+
+  expect(stack).toHaveResourceLike("AWS::CloudFront::Distribution", {
+    DistributionConfig:
+    {
+      Origins: [
+        {
+          OriginPath: "/testPath",
+        }
+      ]
+    }
+  }
```

Comment:
very tiny subjective nit - the same indent for the brace and paren on different lines appears to be wrong (but passes lint). Can we make it like the other locations - on the same line?

```
  });
```

Reply:
To clarify - are you saying you'd like lines 337 and 338 in the same line?

Reply:
Yes